package.json

```
@@ -138,50 +138,50 @@
     "lorem-ipsum": "^2.0.3",
     "luxon": "^1.11.3",
     "moment-timezone": "^0.5.27",
-    "node-sass": "^4.11.0",
-    "postcss-cli": "^6.1.3",
+    "node-sass": "^4.14.0",
+    "postcss-cli": "^7.1.0",
     "prettier": "^1.19.1",
     "pretty-quick": "^2.0.0",
     "puppeteer": "^1.20.0",
-    "react": "16.10.x",
+    "react": "16.13.1",
```

I think this should pull minors at least, to stay somewhat in sync.

```suggestion
    "react": "~16.13.1",
```